crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -654,43 +668,35 @@ impl ScheduleGraph {
 
         // system init has to be deferred (need `&mut World`)
         self.uninit.push((id, 0));
-        self.systems.push(SystemNode::new(config.system));
+        self.systems.push(SystemNode::new(config.item));
         self.system_conditions.push(config.conditions);
 
         Ok(id)
     }
 
     fn configure_sets(&mut self, sets: impl IntoSystemSetConfigs) {
-        let SystemSetConfigs { sets, chained } = sets.into_configs();
-        let mut set_iter = sets.into_iter();
-        if chained {
-            let Some(prev) = set_iter.next() else { return };
-            let mut prev_id = self.configure_set_inner(prev).unwrap();
-            for next in set_iter {
-                let next_id = self.configure_set_inner(next).unwrap();
-                self.dependency.graph.add_edge(prev_id, next_id, ());
-                prev_id = next_id;
-            }
-        } else {
-            for set in set_iter {
-                self.configure_set_inner(set).unwrap();
-            }
-        }
+        self.configure_sets_inner(sets.into_configs(), false);
     }
 
-    fn configure_set(&mut self, set: impl IntoSystemSetConfig) {
-        self.configure_set_inner(set).unwrap();
+    /// Adds the system sets to the graph. Returns a vector of all node ids contained in the nested `SystemnSetConfigs`
```

Comment:
Done. I've renamed the `ancestor_chained` parameter to the more accurate `collect_nodes` name and also fixed the description of the `densely_chained` field a bit, since it describes whether the nested items are linearly chained and not if they should be linearly chained.